recipes/flex/all/test_v1_package/basic_nr.l

```diff
@@ -0,0 +1,89 @@
+/*
+ * This file is part of flex.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions
+ * are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright
+ *    notice, this list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright
+ *    notice, this list of conditions and the following disclaimer in the
+ *    documentation and/or other materials provided with the distribution.
+ *
+ * Neither the name of the University nor the names of its contributors
+ * may be used to endorse or promote products derived from this software
+ * without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED ``AS IS'' AND WITHOUT ANY EXPRESS OR
+ * IMPLIED WARRANTIES, INCLUDING, WITHOUT LIMITATION, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+ * PURPOSE.
+ */
+
+%option C++ noyywrap
```

Comment:
You should need to duplicate these files https://github.com/conan-io/conan-center-index/tree/f706340e542718a9f0ce95baaca8da63dcb11d63/recipes/flex/all/test_v1_package is the same as https://github.com/conan-io/conan-center-index/tree/f706340e542718a9f0ce95baaca8da63dcb11d63/recipes/flex/all/test_package from what I can tell

You can see how it's done in the template https://github.com/conan-io/conan-center-index/blob/f706340e542718a9f0ce95baaca8da63dcb11d63/docs/package_templates/cmake_package/all/test_v1_package/CMakeLists.txt